app/assets/stylesheets/admin/admin.scss

```diff
@@ -47,3 +47,38 @@
 #epiceditor {
```

Comment:
Aahh I didn't realize this file still exists, I plan to delete it. The admin styles are in `app/assets/stylesheets/admin.scss`.

Reply:
oops!